### PR TITLE
Overhaul the env variables needed for the jupyterlab integration

### PR DESCRIPTION
**Pull request by alesgenova**

### CODE REVIEW

File: docker/torchani/.ipynb_checkpoints/Untitled-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.3"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (cjh1) [inline docker/torchani/.ipynb_checkpoints/Untitled-checkpoint.ipynb]: I think this was checked in by mistake

File: docker/torchani/Untitled.ipynb [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.3"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (cjh1) [inline docker/torchani/Untitled.ipynb]: I don't think we want this either

**Comment by cjh1** [comment]
I think that [this](https://github.com/OpenChemistry/mongochemdeploy/blob/master/ansible/roles/docker_compose/templates/docker-compose/env.j2#L33) should be changed to `oc_api_url`?

**Comment by cryos** [comment]
It would be good to get this branch cleaned up, and ready to merge soon. It looks like some useful clean ups, notebooks need removing with a force push/rewrite of commits ideally.

**Comment by cryos** [comment]
It would be good to get this branch cleaned up, and merged soon - maybe target by May 5th?

**Comment by cjh1** [comment]
@alesgenova I would like to revive this. We are going to to have to migrate our NERSC instance to their new infrastructure so that would be a good opportunity to test these changes out. Can you fresh your memory of the changes, it looks like there was a little clean up needed.

**Review by cjh1 — APPROVED** [review]

**Pull request merged**